website/docs/components/icon/partials/guidelines/overview.md

```diff
@@ -0,0 +1,10 @@
+## What is an icon
+
+An icon is used to provide visual flourish to text, and in some rare contexts meaning on its own.
+
+In the context of the HDS components there is an important distinction to make:
+
+- in Figma the term "icon" can be used interchangeably to indicate the Figma component and the visual element itself (or "glyph")
+- in code the "icon" as glyph is nothing more than an `<svg>` HTML element, while the `Hds::Icon` is the actual Ember component that renders the `<svg>` element.
```

Comment:
I'm wondering if this is necessary to say (from line 5 -> 8)? Because even in Figma the icon is also an SVG.

Reply:
I see your point, but it's slightly different, if you think about it: first of all in Figma the glyphs are not `<svg>` but paths (a `.svg` file can be exported, similarly to a `.png`); plus, there is not an `Icon` component, where the content is swapped, but each icon **is** a component (in the [HDS Icons library](https://www.figma.com/design/TLnoT5AYQfy3tZ0H68BgOr/HDS-Icons?t=RMDTpPXI1iCRqwkv-0))

Reply:
That's true, but I'm wondering if saying this will be useful to the consumer. I'll leave others to chime in too.